templates/tools/dockerfile/test/cxx_rbe_jessie_x64/Dockerfile.template

```diff
@@ -0,0 +1,31 @@
+%YAML 1.2
```

Comment:
I think not that many folks know what RBE means -> either add a comment or rename.

Reply:
Changed the name of the file